plugins/osdn/osdn.go

```diff
@@ -0,0 +1,206 @@
+package osdn
+
+import (
+	"fmt"
+	"github.com/golang/glog"
+	"strings"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	kclient "github.com/GoogleCloudPlatform/kubernetes/pkg/client"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/fields"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util/exec"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/watch"
+
+	"github.com/openshift/openshift-sdn/ovssubnet"
+	osdnapi "github.com/openshift/openshift-sdn/pkg/api"
+
+	osclient "github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/sdn/api"
+)
+
+type OsdnRegistryInterface struct {
+	oClient osclient.Interface
+	kClient kclient.Interface
+}
+
+func NetworkPluginName() string {
+	return "redhat/openshift-ovs-subnet"
```

Comment:
This is how it is in the openshift-sdn rpm. Will change this with a separate PR.